gtfs/spec/en/reference.md

```diff
@@ -254,17 +258,21 @@ Primary key (`trip_id`, `stop_sequence`)
+|  `departure_time` | Time | **Conditionally Required** | Departure time from the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`).<br><br>If there are not separate times for arrival and departure at a stop, `arrival_time` and `departure_time` should be the same. <br><br>For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins.<br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for `timepoint=1`.<br>-&nbsp;**Forbidden** when `stop_times.start_pickup_drop_off_window` or `stop_times.end_pickup_drop_off_window` are defined.<br>- Optional otherwise.| |
+|  `stop_id` | Foreign ID referencing `stops.stop_id`, `stop_areas.area_id`, or `id` from `locations.geojson` | **Required** | Identifies the serviced stop. All stops serviced during a trip must have a record in [stop_times.txt](#stop_timestxt). Referenced locations must be stops/platforms, i.e. their `stops.location_type` value must be `0` or empty. A stop may be serviced multiple times in the same trip, and multiple trips and routes may service the same stop.<br><br>If service is on demand, a GeoJSON location or stop area can be referenced:<br>-&nbsp;`id` from `locations.geojson`<br>-&nbsp;`stop_areas.area_id` |
+|  `stop_sequence` | Non-negative integer | **Required** | Order of stops for a particular trip. The values must increase along the trip but do not need to be consecutive.<hr>*Example: The first location on the trip could have a `stop_sequence`=`1`, the second location on the trip could have a `stop_sequence`=`23`, the third location could have a `stop_sequence`=`40`, and so on.* <br><br> Travel within the same stop area or GeoJSON location requires two records in `stop_times.txt` with the same `stop_id` and consecutive values of `stop_sequence`. |
```

Comment:
Actually, do they need to be consecutive or can they be simply increasing as long as there is no stop_sequence in between?

Reply:
Yes, I think this is right. The requirement for stop_sequence should probably just match the language of the core spec ("The values must increase along the trip but do not need to be consecutive.").

Reply:
Thanks for catching this!

Removed "and consecutive values of `stop_sequence`" in `stop_sequence`. It seems original description already cover it so I didn't add any additional sentence. Please let me know if you have other suggestion.